packages/account-v2/src/pages/ConnectedApps/ConnectedAppsResponsive.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { OauthApps } from '@deriv/api-types';
+import { Button } from '@deriv-com/ui';
+import { formatDate, getFormattedAppScopes } from '../../utils/connectedAppsUtils';
+import { ConnectedAppsResponsiveColumn } from './ConnectedAppsResponsiveColumn';
+
+type TConnectedAppsResponsive = {
+    connectedApps: OauthApps;
+    handleToggleModal: (appId: number) => void;
+};
+
+export const ConnectedAppsResponsive = ({ connectedApps, handleToggleModal }: TConnectedAppsResponsive) => (
+    <div className='flex flex-col gap-8'>
+        {connectedApps.map(connectedApp => (
+            <div className='bg-solid-grey-2 rounded-default p-16' key={connectedApp.app_id}>
+                <div className='flex justify-between gap-16'>
+                    <ConnectedAppsResponsiveColumn description={connectedApp?.name} title='Name' />
+                    <ConnectedAppsResponsiveColumn
+                        description={formatDate(connectedApp?.last_used)}
```

Comment:
Can connectedApp be undefined ?

Reply:
Is it possible to get response in this manner where we get an array with objects but one of the objects is undefined? If not then this additional check is redundant 🙏